database/migrations/2020_10_19_151339_add_desc_and_defaultmsg_to_classification.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDescAndDefaultmsgToClassification extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('classifications', function (Blueprint $table) {
+            $table->string('defaultMessage')->nullable();;
+
+            $table->string('description')->nullable();
```

Comment:
I'm assuming we're returning both of these to the frontend, so they'll need to be localized. We're using the [Laravel translatable package](https://github.com/spatie/laravel-translatable) for localizing our model fields, so you'll need to make these columns json instead of string.

This is an example of where we added this type of data before: https://github.com/GCTC-NTGC/TalentCloud/blob/dev/database/migrations/2020_02_24_083132_add_name_and_description_to_job_poster_status.php